tests/x509/test_x509_crlbuilder.py

```diff
@@ -366,6 +366,7 @@ def test_freshestcrl_extension(self, backend):
         ext1 = crl.extensions.get_extension_for_class(x509.FreshestCRL)
         assert ext1.critical is False
         assert isinstance(ext1.value[0], x509.DistributionPoint)
+        assert isinstance(ext1.value[0].full_name, list)
```

Comment:
Does the `is not None` assert work here?

Reply:
yes, **now** it works, after using List as return type annotation (will push in a sec)